g2g_test.go

```diff
@@ -2,26 +2,27 @@ package g2g
 
 import (
 	"expvar"
-	"fmt"
 	"net"
 	"strings"
 	"sync"
 	"testing"
 	"time"
 )
 
+var i = expvar.NewInt("i")
```

Comment:
Hmm. A global variable with the name `i` is certainly not good :) Can you scope this to a specific function? Failing that, can you please give it a much longer name, ideally including the letters `test` at some point?